### DISCUSSION TITLE
Nuxt3 component NuxtImg

### DISCUSSION THREAD

**Discussion by jpa-1330** [nuxt/image#1081] [Q&A]
hi
I have a demo project.(nuxt 3.8.0)
I followed the steps below to install the `Nuxtimg` component.
1.
`pnpm add @nuxt/image`
2.
`export default defineNuxtConfig({
  modules: [
    '@nuxt/image',
  ]
})`
3. add an file(color.jpg) to the public directory 
/public/color.jpg
4.
`<template>
    <NuxtImg src="/color.jpg"/>
</template>`
5.  run the demo
`pnpm run dev`

when the project is running, open the browser.
in the Browser Elements tab
`<img src="/_ipx/_/color.jpg" data-nuxt-img="" srcset="/_ipx/_/color.jpg 1x, /_ipx/_/color.jpg 2x">`

but in the Browser Source code tab,the `_ipx` folder does not exist.
so the image on the page can't be displayed.

Have I done something wrong?
Thank you for your appreciate help.

**Comment by shlomi93** [comment]
I have the same issue, opened an issue and waiting for reply..
